README.md

```diff
@@ -1095,6 +1095,25 @@ source ~/.bashrc
 cd $hotellogs
 ```
 
+## Saving your environment variables
+
+When you export FOO = "BAR" your variable is only exported in this section, to persist in the future you can simply append in your `~/.bash_profile` file the command to export your variable
+```bash
+echo export FOO="BAR" >> ~/.profile
+``` 
+
+## Accessing your scripts
+
+You can easily access your scripts by creating a bin folder in your home with `mkdir ~ / bin`, now all the scripts you put in this folder you can access in any directory.
+
+If you can not access, try append the code below in your `.bash_profile` file and restart your section.
```

Comment:
*section* here doesn't have any sense either, please change it to *shell*, and instead of *restarting* maybe you could just `source` the file.
Also please add the `~` to `.bash_profile`